src/reports/business-intelligence.md

```diff
@@ -17,16 +17,16 @@ Use business intelligence tools to gain the insight you need to make sound busin
 {: .links-table }
 |[Advanced Reporting]({% link reports/advanced-reporting.md %})| Advanced Reporting gives you access to a suite of dynamic reports that are based on your product, order, and customer data, with a personalized dashboard that is tailored to your business needs. This is a cloud-based service that is powered by [Magento Business Intelligence][1]{:target="_blank"}. <br/><br/>For technical information, see [Advanced Reporting][2]{:target="_blank"} in our developer documentation.|
 |[BI Essentials][3]|When you activate a Magento BI Essentials account, you get access to five dashboards with approximately 100 reports. These reports are designed to provide insights around your data and answer questions like “How are my orders growing month-over-month?", "Who are my most loyal customers?, and "Is my coupon strategy working?”|
-|[Yotpo Analytics][6]|[Yotpo][7] is a user-generated content marketing platform enabling merchants to gather, curate, manage, and respond to all kinds of user content from just a single platform. When you have a Yotpo account that is [integrated with your Magento store]({% link marketing/yotpo-connect.md %}), you can access [Yotpo Analytics][6] for reviews, emails, and conversions. <br/><br/>If you have an Yotpo subscription account, you may also use [Yotpo Insights][8] to help understand what your customers are saying about products and gain actionable insights for your business.|
+|[Yotpo Analytics][6]|[Yotpo][7] is a user-generated content marketing platform enabling merchants to gather, curate, manage, and respond to all kinds of user content from just a single platform. When you have a Yotpo account that is [integrated with your Magento store]({% link marketing/yotpo-connect.md %}), you can access [Yotpo Analytics][6] for reviews, emails, and conversions. <br/><br/>If you have a Yotpo subscription account that includes [Yotpo Insights][8], you may also use [Yotpo Insights][9] to help understand what your customers are saying about products and gain actionable insights for your business.|
```

Comment:
```suggestion
|[Yotpo Analytics][6]|[Yotpo][7] is a user-generated content marketing platform enabling merchants to gather, curate, manage, and respond to all kinds of user content from just a single platform. When you have a Yotpo account that is [integrated with your Magento store]({% link marketing/yotpo-connect.md %}), you can access [Yotpo Analytics][6] for reviews, emails, and conversions. <br/><br/>If your Yotpo subscription account includes a [Yotpo Insights plan][8], you may also use [Yotpo Insights][9] to help understand what your customers are saying about products and gain actionable insights for your business.|
```
The repetition was a little confusing for me. Thinking this helps a bit but it is a tricky one.